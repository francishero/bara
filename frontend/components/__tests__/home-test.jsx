import React from 'react';
import renderer from 'react-test-renderer';
import Home from '../home/home';
import {
  Provider,
} from 'react-redux';

it('renders correctly', () => {
  const tree = renderer
    .create(<Provider store={store}><Home /></Provider>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
