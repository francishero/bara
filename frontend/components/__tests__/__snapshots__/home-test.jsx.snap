// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Categories correctly 1`] = `
<div
  className="center home-categories"
>
  <h2>
    Browse Businesses by Category
  </h2>
  <div
    className="category-cards"
  >
    <a
      className="category-card"
      href="#/businesses/?name=&location=New%20York"
    >
      <img
        alt=""
        src={undefined}
      />
      <p>
        Restaurants
      </p>
    </a>
    <a
      className="category-card"
      href="#/businesses/?name=&location=New%20York&tag=nightlife"
    >
      <img
        alt=""
        src={undefined}
      />
      <p>
        Nightlife
      </p>
    </a>
  </div>
</div>
`;
